src/UpdateVisuals.ts

```diff
@@ -430,26 +430,34 @@ export const visualUpdateCubes = () => {
 }
 
 const UpdateHeptGridValues = (type: hepteractTypes) => {
-    const text = type + 'ProgressBarText'
-    const bar = type + 'ProgressBar'
-    const textEl = document.getElementById(text)!
-    const barEl = document.getElementById(bar)!
-    const balance = player.hepteractCrafts[type].BAL
-    const cap = player.hepteractCrafts[type].CAP
-    const barWidth = Math.round((balance / cap) * 100)
-
-    let barColor = '';
-    if (barWidth < 34) {
-        barColor = 'red';
-    } else if (barWidth >= 34 && barWidth < 68) {
-        barColor = '#cca300';
+    const text = type + 'ProgressBarText';
+    const bar = type + 'ProgressBar';
+    const textEl = document.getElementById(text)!;
+    const barEl = document.getElementById(bar)!;
```

Comment:
```suggestion
    const textEl = document.getElementById(text)!;
    const barEl = document.getElementById(bar)!;
```
use DOMCacheGetOrSet